bundler.d/assets.rb

```diff
@@ -1,9 +1,10 @@
 group :assets do
-  gem 'jquery-turbolinks', '~> 2.1'
   gem 'jquery-ui-rails', '< 5.0.0'
   gem 'patternfly-sass', '>= 3.32.1', '< 3.38.0'
   gem 'gettext_i18n_rails_js', '~> 1.0'
   gem 'execjs', '>= 1.4.0', '< 3.0'
   gem 'uglifier', '>= 1.0.3'
   gem 'sass-rails', '>= 5.0', ((RUBY_VERSION < '2.4') ? '< 5.0.8' : '< 6.0')
+  # this one is a dependecy for x-editable-rails
+  gem 'coffee-rails', '~> 5.0.0'
```

Comment:
We need to package this gem as well until we can get rid of it

Reply:
At the moment, 4.2.2 is packaged for RPM. Can we use this for now and update after https://github.com/theforeman/foreman-packaging/pull/4517 got in?

Reply:
we'll still need 5.0 for rails 6 compat

Reply:
That why I said "for now" ;)

Reply:
pushed another commit with both versions dependent on rails version